pom.xml

```diff
@@ -4945,7 +4946,7 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
-          <argLine>-Xmx1g -Dfile.encoding=UTF-8 ${it.failsafe.argLine}</argLine>
+          <argLine>-Xmx1g -Dfile.encoding=UTF-8 @{it.failsafe.argLine}</argLine>
```

Comment:
That's a fix for runtime-binded properties not correctly interpolated if using $ and left empty.